specifyweb/frontend/js_src/lib/components/FormSliders/AttachmentsCollection.tsx

```diff
@@ -25,15 +23,11 @@ export function AttachmentsCollection({
 
   const [scale = defaultScale] = useCachedState('attachments', 'scale');
 
-  const attachments: RA<SerializedResource<Attachment>> = React.useMemo(
+  const attachments = React.useMemo(
     () =>
-      filterArray(
-        collection.models.map((model) => {
-          if (model.specifyModel.name === 'CollectionObjectAttachment') {
```

Comment:
This wasn't correct because it assumed that only collection object attachments are used (many other tables have attachments too)